examples/with-docker/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  /* config options here */
+}
```

Comment:
Is this file required?

Reply:
It is needed for the multistage Dockerfile to build.
I felt this was wise to include as someone pulling the example would expect the Dockerfile to simply work with no modification required

Reply:
Indeed not required

Reply:
When I say its required, this is because the Dockerfile has always contained the following line - https://github.com/zeit/next.js/pull/12879/files#diff-c7b5f2152f889efdbdf9e7615c2cb65bL14

This means `docker build -t next-app -f ./Dockerfile.multistage .` as per the readme notes will fail, as the file `next.config.js` did not previously exist. It is possibly wiser to comment out this line in the Dockerfile as `next.config.js` is not required to run Next.js but people will understand they may need to comment this line back in.

Reply:
Sounds fine!